internal/utils.go

```diff
@@ -0,0 +1,28 @@
+package internal
```

Comment:
It looks like the functions in this file aren't used by anything related to the Metal SSH key refactor; we should still break this change out into a separate refactor, but if it's not necessary for the Metal SSH key refactor, we can remove these changes from the PR now with no impact.